### PR TITLE
cassandra: Add column and table names in parsing error messages

### PR DESCRIPTION
**Pull request by smatting**
This PR adds the column and table names to parsing errors when reading from cassandra query results.

Errors are shown when the `cql` driver cannot read the expected type from the query result, e.g. on an unexpected `null`

```
E, IO Exception occurred, message=cql-io: protocol error: parse error:
response body reading:
Failed reading: withBytes: null (length = -1)
From: Reading column "activated" of table "brig_test.user"  , request=N/A
```

and when the `FromCQl` instance throws an parser error

```
E, IO Exception occurred, message=cql-io: protocol error: parse error:
response body reading:
Failed reading: Unexpected ManagedBy: CqlInt 3
From: Reading column "managed_by" of table "brig_test.user"  , request=N/A
```

This PR uses a new fork of `cql` with a pending MR https://gitlab.com/twittner/cql/-/merge_requests/11

### CODE REVIEW

File: nix/haskell-pins.nix [hunk 1]
```diff
@@ -204,6 +204,13 @@ let
         sha256 = "sha256-htEIJY+LmIMACVZrflU60+X42/g14NxUyFM7VJs4E6w=";
       };
     };
+    cql = {
```
[the remark this answers is not in the document]
Comment (smatting) [reply nix/haskell-pins.nix]: Alright, done

**Comment by akshaymankar** [comment]
Instead of committing the patch here, we can also pull it using this URL https://gitlab.com/twittner/cql/-/merge_requests/11.patch

**Comment by smatting** [comment]
> Instead of committing the patch here, we can also pull it using this URL https://gitlab.com/twittner/cql/-/merge_requests/11.patch

Alright, done

**Comment by elland** [comment]
CI/GH seems to be back to its senses. 👍

**Pull request merged**